packages/query/src/index.ts

```diff
@@ -53,24 +53,44 @@ export class LiveQuery extends TxProcessor implements Storage {
     return true
   }
 
-  private refresh(query: Query): Promise<void> {
-    return this.storage.findAll(query._class, query.query).then(result => query.callback(result))
+  private updateCache<T extends Doc>(query: Query, object: T): void {
```

Comment:
We will have various update logic for different transaction types, so I suggest to rename this to reflect "create object update", such as `updateCreateObject`

Reply:
I suggest `createObjectCaching` like transaction type + 'Caching'? Because for update transaction `updateUpdateObject` so strange)

Reply:
I would be more happy with common prefix, not suffix :) maybe `cacheXXX` like `cacheCreateObject`